src/Angor/Client/Pages/View.razor

```diff
@@ -960,6 +976,73 @@
         }
     }
 
+    private bool hasPendingMessages = false;
+
+    HashSet<string> pubkeyWithMessages = new();
+
+    private async Task FetchPendingSignatures(FounderProject founderProject)
```

Comment:
I don't understand this code really, we just need to find out if there are pending messages 

All you need to do here is call the method
```
        await RelayService.LookupSignaturesDirectMessagesForPubKeyAsync(FounderProject.ProjectInfo.NostrPubKey, FounderProject.LastRequestForSignaturesTime?.AddSeconds(1), 1,
            _ =>
            {
                if (InvestmentRequests)
                    return;

                InvestmentRequests = true;
                StateHasChanged();
            });
```

and if there are InvestmentRequests  then mark the button